libminifi/test/script-tests/PythonExecuteScriptTests.cpp

```diff
@@ -29,6 +29,15 @@
 #include "processors/GetFile.h"
 #include "processors/PutFile.h"
 
+// ,-------------------------------------------------------------,
+// | ! |                      Disclaimer                     | ! |
+// |---'                                                     '---'
+// |                                                             |
+// | This file contains tests for the "ExecuteScript" processor, |
+// |         not for the "ExecutePython" processor.              |
+// |                                                             |
+// '-------------------------------------------------------------'
```

Comment:
Ah, I liked this thought, it reminded me that I am looking at the wrong file when checking tests :) Will replace this and rename the file to an ugly-long-named-one instead so that it will be harder to confuse this file With `ExecutePythonProcessorTests.cpp` :)

Reply:
Renamed the tests.